docs/components/card.md

```diff
@@ -74,6 +74,16 @@ String to be used as a title. Can also be [slotted](#slots).
 <KCard title="Card Title" />
 ```
 
+### titleTag
+
+HTML element you want title to be rendered as. Defaults to `div`.
+
+<KCard title-tag="h4" title="Card Title H4 Heading" />
```

Comment:
question: Why wouldn't this just be accomplished via the slot?

Reply:
Because a slot would require extra 3 lines instead of just 1 and because this gives more flexibility without a burden of being too opinionated